test/QtDBMigrationTest.pro

```diff
@@ -18,13 +18,3 @@ include("../src/QtDBMigration.pri")
 
 SOURCES += \
     QtDBMigrationTest.cpp
-
-INCLUDEPATH += \
```

Comment:
This should not be removed (that is why you had to fix that `#include`).

Reply:
Restored back to its original state.